src/Stratis.Bitcoin.Features.Consensus/CoinViews/CachedCoinView.cs

```diff
@@ -234,7 +235,7 @@ private CachedCoinView(IDateTimeProvider dateTimeProvider, ILoggerFactory logger
         }
 
         /// <summary>
-        /// Finds all changed records in the cache and persists them to the underlaying coinview.
+        /// Finds all changed records in the cache and persists them to the underlying CoinView.
```

Comment:
Why `CoinView` instead of `coinview`? If you want to reference an object, you're supposed to use `<see>`, but I don't think that was intended here

Reply:
yes I will change this one back agreed